OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -140,17 +191,35 @@ $ sudo chown 1000:1000 /scratch/wcpdocker/volumes/es
 #### B. Creating and Running Elasticsearch Container
 
 ```
-$ docker run -i -t --name ESContainer --network=WCPortalNET -p 9200:9200 --volumes-from WCPortalContainer -v $ES_DATA_VOLUME:/u01/esHome/esNode/data --env-file <directory>/webcenter.env.list $WCPortalImageName configureOrStartElasticsearch.sh
+$ docker run -i -t --name <ES_CONTAINER_NAME> --network=<NETWORK_NAME> -p <HostFreePort>:9200 --volumes-from <WCP_CONTAINER_NAME> -v <ES_DATA_VOLUME>:/u01/esHome/esNode/data --env-file webcenter.env.list oracle/wcportal:12.2.1.4.0 configureOrStartElasticsearch.sh
 ```
+Sample Run Command
```

Comment:
Please add a blank line before and after this line and change it to "Sample run command:"